script/plume-front.sh

```diff
@@ -1,8 +1,24 @@
 #!/bin/bash
 
-ARCH=`arch`
+ARCH=$(python <<EOF
```

Comment:
Wouldn't it be possible to do the same with Bash? And maybe you could use Python 3 if it present instead of importing from `__future__`.

Reply:
> Wouldn't it be possible to do the same with Bash?

yes. with great difficulty.

> And maybe you could use Python

+1